### ISSUE TITLE
Error when opening app switcher ("Can't create control.")

### ISSUE THREAD

**Issue by 1j01** [1j01/window-switcher#1]
I don't know what causes this. It looks like it's failing to create the Pic component (image) for the focus ring.
Reloading the script fixes it, but I've seen this happen a few times.
```
Error: Can't create control.

	---- C:\Program Files (x86)\Window Switcher\GuiEnhancerKit.ahk
	000: }
	135: {
▶	135: Return super.Add(ControlType, Options?, Text?)
	135: }
	137: {

Call stack:
C:\Program Files (x86)\Window Switcher\GuiEnhancerKit.ahk (135) : [Gui.Prototype.Add] Return super.Add(ControlType, Options?, Text?)
C:\Program Files (x86)\Window Switcher\GuiEnhancerKit.ahk (135) : [GuiExt.Prototype.Add] Return super.Add(ControlType, Options?, Text?)
C:\Program Files (x86)\Window Switcher\app-switcher.ahk (187) : [ShowAppSwitcher] FocusRing := AppSwitcher.Add("Pic", "yM w128 h128 Section", ResourcesDir "app-border-inactive.png")
C:\Program Files (x86)\Window Switcher\app-switcher.ahk (360) : [<Hotkey>] ShowAppSwitcher(Apps)
> #Tab
```
